content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,411 @@
+---
+title: SPIRE
+description: Describes how to configure Istio to integrate with SPIRE to get cryptographic identities through Envoy's SDS API.
```

Comment:
Nit: The other headings are with `How to ...` or `Information on how to ...`. Maybe change this to the former by removing Describes?